libraries/loader/src/loader.js

```diff
@@ -158,9 +157,16 @@ function loader(componentDirectory, virtualComponents = {}) {
       }
       return loaded[target];
     }
-
-    return load(target);
+    return recursiveLoad(target);
+  };
+  //method to crash the process if error occured
+  load.crashOnError = function(target) {
+    load(target).catch(err => {
+      console.log(err.stack);//will print the error
```

Comment:
This comment is redundant -- it says the same thing as the code on this line.  Please remove :)